v5.1.1.sql

```diff
@@ -323,3 +323,15 @@ ALTER TABLE openchpl.certified_product DROP COLUMN IF EXISTS terms_of_use_url;
 ALTER TABLE openchpl.pending_certified_product DROP COLUMN IF EXISTS terms_of_use_url;ALTER TABLE openchpl.test_tool add column retired boolean NOT NULL DEFAULT false;
 UPDATE openchpl.test_tool set retired = true where name = 'Transport Testing Tool';
 UPDATE openchpl.test_tool set retired = true where name = 'Transport Test Tool';
+
```

Comment:
Removed changes in v5.1.1.sql and added a line for the insert in openchpl_preload. Also removed the old insert at the bottom of openchpl_preload.sql